### PR TITLE
Week1-

### PR DESCRIPTION
**Pull request by AnujaBujurge29**

### CODE REVIEW

File: 01-node-tutorial/answers/prompter.js [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-    const bodyArray = body1.split("&");
-    const resultHash = {};
-    bodyArray.forEach((part) => {
-      const partArray = part.split("=");
-      resultHash[partArray[0]] = partArray[1];
-    });
-    callback(resultHash);
-  });
-};
+// const getBody = (req, callback) => {
```
Comment (ethanarns) [inline 01-node-tutorial/answers/prompter.js:4]: I think you may have forgotten to do something here. Commenting this out unfortunately makes it break and not compile.

File: 01-node-tutorial/answers/11-fs-async.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const { readFile, writeFile } = require('fs')
+
+readFile('../content/first.txt', 'utf8', (err, result) => {
+    if (err) {
+        console.log(err);
+        return
+    }
+    // console.log(result);
+    const first = result;
+    readFile('../content/second.txt', 'utf8', (err, result) => {
```
Comment (ethanarns) [inline 01-node-tutorial/answers/11-fs-async.js:10]: Good work! You got it!

**Review by ethanarns — COMMENTED** [review]
Unfortunately, I think you forgot to edit prompter.js here. It does not compile because a big portion of it was removed via commenting. However, the other fixes are good!

**Comment by AnujaBujurge29** [comment]
Hi,
Thanks for your review. Please find new Pull request:

https://github.com/AnujaBujurge29/node-express-course/pull/3

Let me know if we still have errors.

Thank you for your guidance.
Anuja




[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
03/15/24,
09:38:06 AM

**Pull request merged**